spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/AwsS3EnvironmentRepository.java

```diff
@@ -68,45 +72,53 @@ public void setOrder(int order) {
 
 	@Override
 	public Environment findOne(String specifiedApplication, String specifiedProfiles, String specifiedLabel) {
-		final String application = StringUtils.isEmpty(specifiedApplication)
+		final String application = ObjectUtils.isEmpty(specifiedApplication)
```

Comment:
Why the change from StringUtils to ObjectUtils?

Reply:
It was deprecated so I figured I would just switch it while I was in the file

Reply:
right